### PR TITLE
created okv monitor corrected structure for integ

### PR DESCRIPTION
**Pull request by mfioramo**
please review

### CODE REVIEW

File: manageability-and-operations/operations-advisory/okv-monitoring/files/main.go
Comment (AlexanderHodicke) [inline manageability-and-operations/operations-advisory/okv-monitoring/files/main.go:1]: @oheimburger please review

**Review by AlexanderHodicke — COMMENTED** [review]
A code review by @oheimburger is needed. The rest looks fine.

File: manageability-and-operations/operations-advisory/okv-monitoring/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+It dumps all the snmp  info from any OKV as described in the documentation:
+https://docs.oracle.com/en/database/oracle/key-vault/21.1/okvag/monitoring.html#GUID-C695162D-1BB4-4B86-889E-24322FD66EE7 
+
+# Instructions
+
+1. You need to install go 
+2. Copy main.go file to any directory
+3. Execute go mod init
+4. Execute go mod tidy
+5. Run the program as go run main.go -target="okv_ip" -Port=161 -UserName="okv_snmp_user" -AuthenticationPassphrase="okv_snmp_user" -PrivacyPassphrase="okv_snmp_user"
```
Comment (oheimburger) [inline manageability-and-operations/operations-advisory/okv-monitoring/README.md:13]: Make sure that command line can be distinguished from normal text, like "run ```go run main.go```"

File: manageability-and-operations/operations-advisory/okv-monitoring/files/main.go [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+	params := &g.GoSNMP{
+		Target:        "10.0.0.190",
+		Port:          161,
+		Version:       g.Version3,
+		SecurityModel: g.UserSecurityModel,
+		MsgFlags:      g.AuthPriv,
+		Timeout:       time.Duration(30) * time.Second,
+		SecurityParameters: &g.UsmSecurityParameters{UserName: *UserNamePtr,
+			AuthenticationProtocol:   g.SHA,
+			AuthenticationPassphrase: "Welcome123.",
```
Comment (oheimburger) [inline manageability-and-operations/operations-advisory/okv-monitoring/files/main.go:81]: Password! Branch needs to be deleted.

File: manageability-and-operations/operations-advisory/okv-monitoring/files/main.go [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+			PrivacyProtocol:          g.AES,
+			PrivacyPassphrase:        "Welcome123.",
```
Comment (oheimburger) [inline manageability-and-operations/operations-advisory/okv-monitoring/files/main.go:83]: Password! Branch needs to be deleted.

File: manageability-and-operations/operations-advisory/okv-monitoring/files/main.go [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+	//err = params.BulkWalk(".1.3.6.1.4.1.2021.2", printValue)
+	//err = params.BulkWalk(".1.3.6.1.4.1.2021.2.1.2.3", printValue)
+	// err = params.BulkWalk(".1.3.6.1.4.1.2021.2.1.2", printValue)
+	//values,err = params.BulkWalkAll(".1.3.6.1.4.1.2021.2.1")
+	if err != nil {
+		fmt.Printf("Walk Error: %v\n", err)
+		os.Exit(1)
+	}
+	
+	/*
```
Comment (oheimburger) [inline manageability-and-operations/operations-advisory/okv-monitoring/files/main.go:233]: Commented code should not be present in GitHub. Remove.

File: manageability-and-operations/operations-advisory/okv-monitoring/files/main.go [hunk 5]
```diff
@@ -0,0 +1,283 @@
...
+	// sysServices represents the value that indicates the set of services primarily offered. The value is a sum.
+	 fmt.Printf("represents the value that indicates the set of services primarily offered. The value is a sum.\n")	 
+	 err = params.BulkWalk("1.3.6.1.2.1.1.7", printValue)	 
+	fmt.Printf("*------------------------------*\n")
+	 	 	 
+	 	 
+	 
+	 
+	
+	//err = params.BulkWalk(".1.3.6.1.4.1.2021.2", printValue)
```
Comment (oheimburger) [inline manageability-and-operations/operations-advisory/okv-monitoring/files/main.go:224]: Commented code should not be present in GitHub. Remove.

File: manageability-and-operations/operations-advisory/okv-monitoring/files/main.go [hunk 6]
```diff
@@ -0,0 +1,283 @@
...
+	
+	/*
+	fmt.Printf("len=>%d\n", len(values))
+	
+	for i, item := range values {
+	 fmt.Printf("values[%d] %s\n", i, item.Name)
+	}
+*/	
+
+/*
```
Comment (oheimburger) [inline manageability-and-operations/operations-advisory/okv-monitoring/files/main.go:241]: Commented code should not be present in GitHub. Remove.

File: manageability-and-operations/operations-advisory/okv-monitoring/files/main.go [hunk 7]
```diff
@@ -0,0 +1,283 @@
+// a very simple snmp okv crawler
```
Comment (oheimburger) [inline manageability-and-operations/operations-advisory/okv-monitoring/files/main.go:1]: Use common English grammar and punctuation!

**Review by oheimburger — CHANGES_REQUESTED** [review]
Code contains passwords and branch must be deleted.
Before submitting a new version, review the other comments, too.

**Pull request closed without merging**